README.md

```diff
@@ -13,7 +13,7 @@ Bevy is a refreshingly simple data-driven game engine built in Rust. It is free
 
 ## WARNING
 
-Bevy is still in the _very_ early stages of development. APIs can and will change (now is the time to make suggestions!). Important features are missing. Documentation is sparse. Please don't build any serious projects in Bevy unless you are prepared to be broken by API changes constantly.
+Bevy is still in the early stages of development. Important features are missing. Documentation is sparse. And minor versions of Bevy containing breaking changes to the API are released [approximately once every 3 months](https://bevyengine.org/news/bevy-0-6/#the-train-release-schedule). Use at your own risk.
```

Comment:
```suggestion
Bevy is still in the early stages of development. Important features are missing. Documentation is sparse. A new version of Bevy containing serious breaking changes to the API is released [approximately once every 3 months](https://bevyengine.org/news/bevy-0-6/#the-train-release-schedule). Use at your own risk.
```

Reply:
They are technically minor versions, but I think that carries the wrong connotation.